distributionviewer/core/static/css/logout.styl

```diff
@@ -1,7 +1,16 @@
 @import 'lib';
 
 .sign-out-wrapper {
+  .email {
```

Comment:
Since the `email` class doesn't appear anywhere else at the moment, we could move this out of the `.sign-out-wrapper` parent to reduce specificity.